api/envoy/config/listener/v3/listener.proto

```diff
@@ -32,13 +32,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Listener configuration]
 // Listener :ref:`configuration overview <config_listeners>`
 
+// The additional address the listener is listening on.
+// [#not-implemented-hide:]
+message AdditionalAddress {
+  repeated core.v3.Address address = 1;
```

Comment:
I don't think this should be `repeated`. This message is already `repeated` on line 122.

Reply:
yes, sorry about that, I forget to remove it.